iml-agent/Cargo.toml

```diff
@@ -47,6 +47,7 @@ iml-fs = { path = "../iml-fs", version = "0.1.0" }
 [dev-dependencies]
 mockito = "0.20.0"
 pretty_assertions = "0.6.1"
+insta = "0.11.0"
```

Comment:
Use `0.11` here so patch changes are pulled in.

Reply:
👍